src/components/GuestInputForm/GuestInputForm.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useUser } from '../../context/UserContext';
+import { useEntires } from '../../context/EntriesContext';
+import style from './GuestInputForm.css';
+import { useState } from 'react';
+
+export default function GuestInputForm() {
+  const [name, setName] = useState('');
+  const [guestEntry, setGuestEntry] = useState('');
+  const { user, setUser } = useUser();
+  const { entries, setEntries } = useEntires();
+
+  const handleUpdating = () => {
+    setUser(name);
+    setEntries([
+      ...entries,
+      { name: name, entry: guestEntry, id: entries.length },
```

Comment:
`entries.length` is the equivalent of giving it's index as an id. We should avoid this in React